rs/registry/canister/canister/registry.did

```diff
@@ -480,7 +488,7 @@ service : {
   change_subnet_membership : (ChangeSubnetMembershipPayload) -> ();
   clear_provisional_whitelist : () -> ();
   complete_canister_migration : (CompleteCanisterMigrationPayload) -> ();
-  create_subnet : (CreateSubnetPayload) -> ();
+  create_subnet : (CreateSubnetPayload) -> (CreateSubnetResponse);
```

Comment:
is this breaking?

Reply:
I believe this is ok. I've added an experiment to the PR description that says `didc` considers this to be a safe evolution. (Also, there is a compatibility check that would block this PR if Candid thought this is not compatible.) Also, I will write an end to end test.